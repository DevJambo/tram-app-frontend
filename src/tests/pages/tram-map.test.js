import React from "react";
import { render } from "@testing-library/react";
import TramMap from "../../components/pages/tram-map";

describe("HomePage", () => {
  const { asFragment } = render(<TramMap />);

  it("renders correctly", () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
