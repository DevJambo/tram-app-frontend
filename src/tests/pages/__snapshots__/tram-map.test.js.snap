// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h2
      class="map-header"
    >
      Tram Map Page
    </h2>
  </div>
</DocumentFragment>
`;
